include/wabt/ir.h

```diff
@@ -1280,6 +1281,9 @@ struct Module {
     bool exceptions = false;
     bool threads = false;
   } features_used;
+
+  // wasm2c uses this information to limit its output.
```

Comment:
I wonder if we should be more specific here.   How about `funcs_in_table_elems`?   Or `required_func_wrappers`?

And perhaps a more specific comment such as "// wasm2c needs to generate small wrapper functions for any functions used in table elements.   We keeps track of them here so we can emit them in WriteFuncRefWrappers"

Reply:
they can be used in table, global, elem initializers or (if used in any of the above, or apparently if exported?) function code. so it's not just table elements that we need to worry about.

we don't like the idea of putting `required_func_wrappers` in `ir.h` since `ir.h` is more general than wasm2c.

Reply:
Oh I see.  In that case I guess just update the comment mentioning why wasm2c needs to track this info.

Reply:
done